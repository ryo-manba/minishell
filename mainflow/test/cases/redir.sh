> tmpfile1 | file tmpfile1
>> tmpfile2 | file tmpfile2
echo hello > tmpfile1 | echo world > tmpfile2
cat tmpfile1 tmpfile2
cat < tmpfile1
echo hello > tmpfile2 >> tmpfile1
cat tmpfile1
echo hello > tmpfile1 | echo hello >> tmpfile1
cat < tmpfile1
echo hello > tmpfile1 > tmpfile2 > tmpfile3 | file tmpfile1 tmpfile2 tmpfile3 | rm tmpfile1 tmpfile2 tmpfile3
aaa > tmpfile1 | bbb > tmpfile2 | ccc > tmpfile3 | file tmpfile1 tmpfile2 tmpfile3 | rm tmpfile1 tmpfile2 tmpfile3
echo hello 1>tmpfile1 2>tmpfile2 | cat tmpfile1 | rm tmpfile1 tmpfile2
echo hello 1>tmpfile1 10240>tmpfile2  2>tmpfile3 | file tmpfile1 tmpfile2 tmpfile3 | rm tmpfile1 tmpfile2 tmpfile3
echo hello 1>tmpfile1 2147483648>tmpfile2 > tmpfile3 | cat tmpfile1 | rm tmpfile1 tmpfile2 tmpfile3
