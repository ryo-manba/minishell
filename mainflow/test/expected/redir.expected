tmpfile1: empty
tmpfile2: empty
hello
world
hello
hello
hello
hello
hello
test/cases/redir.sh: line 11: aaa: command not found
test/cases/redir.sh: line 11: bbb: command not found
test/cases/redir.sh: line 11: ccc: command not found
test/cases/redir.sh: line 13: 10240: Bad file descriptor
rm: tmpfile3: No such file or directory
test/cases/redir.sh: line 14: file descriptor out of range: Bad file descriptor
rm: tmpfile3: No such file or directory
